src/main/java/eu/europeana/metis/sandbox/service/dataset/DatasetReportServiceImpl.java

```diff
@@ -116,10 +121,14 @@ public ProgressInfoDto getReport(String datasetId) {
         recordsProcessedByStep.forEach((step, statusMap) -> addStepInfo(stepsInfo, statusMap, step,
                 recordErrorsByStep));
 
+        // prepare object TiersZeroInfo
```

Comment:
comment is not necessary, the method already is explaning what is doing.

Reply:
Removed